src/test_driver/openiotsdk/unit-tests/test_components.txt

```diff
@@ -18,4 +18,5 @@ SupportTests
 UserDirectedCommissioningTests
 SecureChannelTests
 ICDServerTests
+ControllerTests
```

Comment:
previously it seems controllertests were not part of openiot (not sure why) ... since openiot fails, should we add them here?

If they cannot run on openiot, it would be nice to add that info to BUILD.gn

Reply:
Ok.  I've removed ControllerTests from test_components.txt